src/main/java/frc/robot/RobotContainer.java

```diff
@@ -98,6 +105,10 @@ public RobotContainer() {
                     return nav.getTargetHeading();
                 });
 
+        drive.setTargetHeadingSupplier(
+                () -> {
+                    return nav.getTargetHeading();
+                });
```

Comment:
It looks like this got duplicated in the merge conflict resolution

Reply:
removed